sway/commands/output/background.c

```diff
@@ -117,23 +117,13 @@ struct cmd_results *output_cmd_background(int argc, char **argv) {
 		}
 
 		bool can_access = access(src, F_OK) != -1;
-		if (!can_access) {
-			sway_log_errno(SWAY_ERROR, "Unable to access background file '%s'",
```

Comment:
This error message, and maybe also the swaynag warning, would be useful to keep in the case where the file cannot be accessed and there there is a fallback color.